content/en/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -82,7 +82,7 @@ replace the `containerd-shim-runhcs-v1.exe` is installed at `$Env:ProgramFiles\c
 
 ## Limitations
 
-- HostProcess containers require version 1.5.4 or higher of the containerd {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
+- HostProcess containers require the latest version of containerd {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}.
```

Comment:
This should be specific version. "latest" is to arbitrary

Reply:
Changed to 1.6 or higher